### ISSUE TITLE
Build fails with 1.8.0 - Works with 1.7.2

### ISSUE THREAD

**Issue by faustbrian** [mrmartineau/gatsby-theme-code-notes#23]
Hey, looking at https://github.com/mrmartineau/gatsby-theme-code-notes/compare/v1.7.2...v1.8.0 I can't see any obvious breaking changes from 1.7.2 to 1.8.0 but building on multiple projects now fails with the below error message.

```
failed Building static HTML for pages - 1.601s

 ERROR #95313

Building static HTML failed for path "/"

See our docs page for more info on this error: https://gatsby.dev/debug-html

  WebpackError: ReferenceError: name is not defined

  - build-html.js:107 doBuildPages
    /[gatsby]/dist/commands/build-html.js:107:24

  - build-html.js:121 async buildHTML
    /[gatsby]/dist/commands/build-html.js:121:3

  - build.js:206 async build
    /[gatsby]/dist/commands/build.js:206:5
```

Where there any breaking changes that I overlooked that would cause this?

**Comment by faustbrian** [comment]
`yarn build` seems to be also broken on a fresh install.

**Comment by mrmartineau** [comment]
@faustbrian 🤦‍♂️ was busy today after I released v1.8.0, have found the culprit and will release a fix shortly.

**Comment by mrmartineau** [comment]
hey @faustbrian, I have just published v1.9.0 which fixes this issue

**Comment by faustbrian** [comment]
Thanks for the fix 👍
